packages/@webex/test-helper-mock-webex/src/index.js

```diff
@@ -282,6 +282,7 @@ function makeWebex(options) {
     feature: {
       setFeature: sinon.stub().returns(Promise.resolve(false)),
       getFeature: sinon.stub().returns(Promise.resolve(false)),
+      setMercury: sinon.stub(),
```

Comment:
I think we can remove this as we have changed the approach.

Reply:
Oops. Thanks for noticing. Removed